app/cdap/components/WranglerGrid/SelectColumnPanel/ColumnsList/index.tsx

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import { NoDataSVG } from 'components/GridTable/iconStore';
+import T from 'i18n-react';
+import { ISelectColumnsListProps } from 'components/WranglerGrid/SelectColumnPanel/ColumnsList/types';
+import { IHeaderNamesList } from 'components/WranglerGrid/SelectColumnPanel/types';
+import ColumnTable from 'components/WranglerGrid/SelectColumnPanel/DataTable';
+import { MULTI_SELECTION_COLUMN } from 'components/WranglerGrid/SelectColumnPanel/constants';
+import SelectedColumnCountWidget from 'components/WranglerGrid/SelectColumnPanel/CountWidget';
+import { IMultipleSelectedFunctionDetail } from 'components/WranglerGrid/SelectColumnPanel/types';
+import { SELECT_COLUMN_LIST_PREFIX } from 'components/WranglerGrid/SelectColumnPanel/constants';
+import { NormalFont, SubHeadBoldFont } from 'components/common/TypographyText';
+import { Box, IconButton } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import styled from 'styled-components';
+import { grey } from '@material-ui/core/colors';
+import {
+  getColumnsSupportedType,
+  getFilteredColumn,
+} from 'components/WranglerGrid/SelectColumnPanel/utils';
+
+const SearchIconComponent = styled(SearchIcon)`
+  &.MuiSvgIcon-root {
```

Comment:
if you are styling an icon plz use something more conventional
SearchIconComponent doesn't make sense
moreover, we should not use component keyword for naming the components 

plz make it StyledSearchIcon or something better

Reply:
Done